core/src/core/temporal/activities.py

```
@@ -0,0 +1,534 @@
+        return len(result.response)
+
+    if hasattr(result.response, "__len__"):
+        return len(result.response)
+
+    return 1
+
+
+def _get_polars_dataframe() -> type["PolarsDataFrame"] | None:
+    """Ленивый импорт Polars DataFrame."""
+    try:
+        from polars import DataFrame  # noqa: PLC0415
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)